website/docs/r/iam_role_policy.html.markdown

```diff
@@ -72,3 +72,11 @@ assign a random, unique name.
 * `name` - The name of the policy.
 * `policy` - The policy document attached to the role.
 * `role` - The role to which this policy applies.
+
+## Import
+
+IAM Role Policies can be imported using the `name`, e.g.
+
+```
+$ terraform import aws_iam_role_policy.mypolicy mypolicy_name
```

Comment:
It looks like we are missing a bit more info there, as you can see in the [code](https://github.com/terraform-providers/terraform-provider-aws/blob/master/aws/resource_aws_iam_role_policy.go#L80): the ID for this resource is constructed as `<role name>:<policy name>`.